content/en/news/spanish-traditional-chinese-available.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Spanish and Traditional Chinese Localization Available in KubeSphere Web Console'
+createTime: '2020-07-27'
+keywords: KubeSphere, Localization, Community Contribution
+description: The web console of KubeSphere v3.0 now supports the localization of Spanish and Traditional Chinese.
+author: 'Feynman, Sherlock'
+---
+
+The web console of KubeSphere v3.0 now supports the localization of Spanish and Traditional Chinese. In User Settings, you can select the language of web console in the drop-down menu.
+
+![KubeSphere-language-setting](https://ap3.qingstor.com/kubesphere-website/docs/KubeSphere-language-setting.png)
+
+Here, we would like to thank Geko, our Spanish partner that provides the Spanish localization. This serves as a great start for the cooperation between KubeSphere and Geko as the two sides promise to further develop the Spanish community. With a brand-new Spanish web console, KubeSphere aims to create better user experiences for users around the world who prefer Spanish UI.
```

Comment:
I recommend you to add a hyperlink for the organization or people when you mention them in a post, they would be happy to see your introduction to their homepage or social media account.

Example: we would like to thank [Geko Cloud](https://geko.cloud/).